Tools/adb_upload_to_bebop.sh

```diff
@@ -34,11 +34,25 @@ adb shell mount -o remount,rw /
 adb shell touch /home/root/parameters
 adb shell mkdir -p /data/ftp/internal_000/fs/microsd
 
-${RPI_TOOLCHAIN_DIR}/gcc-linaro-arm-linux-gnueabihf-raspbian/bin/arm-linux-gnueabihf-strip \
+# kill PX4 if it is already running from autostart
+restart_px4=false
+adb_return=$(adb shell killall -KILL px4)
+if [[ $adb_return == "" ]]; then
+    echo "Killed running PX4 process"
+    restart_px4=true
+fi
+
+`dirname ${PX4_CXX_COMPILER}`/arm-linux-gnueabihf-strip \
   -R .comment -R .gnu.version \
   ../build_posix_bebop_default/src/firmware/posix/px4
```

Comment:
Kind of like this (https://github.com/PX4/Firmware/blob/master/src/firmware/posix/sitl_tests.cmake#L59), but with add_custom_command (https://cmake.org/cmake/help/v3.0/command/add_custom_command.html).